about.html

```diff
@@ -12,53 +12,86 @@
 </head>
 
 <body>
-    <div class="container about-page">
-     
-    About
-
-    Leverage agile frameworks to provide a robust synopsis for high level overviews. Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition. Organically grow the holistic world view of disruptive innovation via workplace diversity and empowerment.
-
-
-    <img src="img/about-plan.png" alt="strategy">
-    
-    Strategy
     
-    Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.
-
-    Learn More
-
-
-
-    <img src="img/about-working.png" alt="strategy">
-    
-    How We Work
+    <div class="container about-page">
     
-    Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.
+      <section class="heading">
+       <nav class="heading-nav">
+          <img src="./img/lambda-black.png" alt="Lambda Logo Black">
+          <a href="#">Home</a>
+          <a href=".\about.html">About</a>
```

Comment:
Great job linking the about page to the navbar, adding the ./index.html to the home button will add more functionality and allow users to go between both pages.